Dockerfile.worker

```diff
@@ -14,6 +14,11 @@ COPY data_models/dist/data-refinery-models-* data_models/
 # Get the latest version from the dist directory.
 RUN pip install data_models/$(ls data_models | sort -r | head -1)
 
+COPY data_models/dist/data-refinery-models-* data_models/
+
+# Get the latest version from the dist directory.
+RUN pip install data_models/$(ls data_models | sort -r | head -1)
```

Comment:
Hmm, I'm not sure why but `sort` was able to handle all the file names being on the same line somehow. However `ls -1 | tail -1` is certainly better so I'll use that.